docs/build/p2p-node/index.md

```diff
@@ -1,25 +1,19 @@
 ---
 id: p2p-node
 order: 3
-title: P2P Node
-contributors: <Dennis Wellmann:dennis@centrifuge.io>
```

Comment:
I think we can keep a list of contributors when a major refactor happened, so I would keep dennis here too.

Reply:
Done. Thank you!